earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -61,6 +64,8 @@ main_postinstall()
     chmod 755 "$BININSTALLROOTDIR/gefusion"
     chmod 755 "$BININSTALLPROFILEDIR/ge-fusion.csh"
     chmod 755 "$BININSTALLPROFILEDIR/ge-fusion.sh"
+    chown root:$GEGROUP $BASEINSTALLDIR_VAR/run
```

Comment:
I know `GEGROUP` is not supposed to contain any special characters, but I still like quotes. `BASEINSTALLDIR_VAR` may be less guaranteed to be safe.

Reply:
I can add quoting.